app/models.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from flask import Flask
+
+
+@dataclass
+class WhorfConfig:
+    ignores_namespaces: list[str]  # a list of namespaces to ignore requests from
+    upload_interval_in_min: str = "*/5"  # every 5 minutes
```

Comment:
@gruebel 1 hour should be enough IMO

Reply:
Also, perhaps we should report more things than just the results - let's talk with @tsmithv11 on that

Reply:
maybe I was not so clear about the upload results, it not only uploads the pure results, but also persists the repo and run metadata. So, the same as we do in `checkov`.

Reply:
Yeah yeah. Still, every 5 minutes is a lot. And perhaps other metrics such as # of webhooks called etc is also interesting

Reply:
the number of webhooks is directly related to how many files are scanned in the folder.
yeah let's go with a higher interval, I just used one which is not too often, but also not so rare to better demo it 🙂